packages/fast-style/src/font.ts

```diff
@@ -1,7 +1,12 @@
 import { combine, Enum } from '@rnw-community/object-field-tree';
 
-export const getFont = (fontFamilyObj: Enum, fontSizeObj: Enum, fontColorObj: Enum) =>
-    combine(
+export const getFont = (fontFamilyObj: Enum, fontSizeObj: Enum<string>, fontColorObj: Enum) => {
```

Comment:
@macseem Change to `fontSizeObj: Enum<string | number>`

Reply:
changed